source/modules/MEI.critapp.xml

```diff
@@ -63,9 +63,9 @@
     <remarks>
       <p>The alternatives provided in <gi scheme="MEI">lem</gi> and/or <gi scheme="MEI">rdg</gi>
         sub-elements may be thought of as exclusive or as parallel. The <att>type</att> attribute
-        may contain any convenient descriptive word, describing the extent of the variation (e.g.
-        note, phrase, measure, etc.), its text-critical significance (e.g. significant, accidental,
-        unclear), or the nature of the variation or the principles required to understand it (e.g.
+        may contain any convenient descriptive word, describing the extent of the variation (e.g.,
```

Comment:
I think they were missed here?

Reply:
Sorry, I should have been more clear. I added `abbr` tags in the docs for the Guidelines only, not in the schema files itself. I wasn't sure if this would be needed or even a good idea. If you say so, I'll add them there, too.

Reply:
I don't see why not. The only reason to not do it is if the TEI markup isn't supported in this context, but I don't think that's the case.

Reply:
Done.